docs/src/main/asciidoc/tika-guide.adoc

```diff
@@ -48,11 +48,22 @@ This will add the following to your pom.xml:
 [cols="<m,<m,<2",options="header"]
 |===
 |Property Name|Default|Description
-|quarkus.tika.tika-config-path||The resource path within the application artifact to the https://tika.apache.org/1.22/configuring.html[Tika configuration] resource. Typically a file named `tika-config.xml` is added to the root of the application resources path. The default configuration will be used if no configuration resource is available.
-|quarkus.tika.parsers||Comma-separated list of the abbreviated or full parser class names which have to be loaded by the extension. A `pdf` abbreviation can be used to refer to the PDF parser and custom abbreviations must be used for all other parsers. Note that this property is mutually exclusive with the `tika-config-path` property. 
+|quarkus.tika.parsers||Comma-separated list of the abbreviated or full parser class names which have to be loaded by the extension. Currently `pdf` and `odf` abbreviations can be used to refer to PDF and OpenDocument format parsers respectively and custom abbreviations must be used for all other parsers. Note that this property is mutually exclusive with the `tika-config-path` property and should be be preferred when the individual parsers have to be configured.  
 |quarkus.tika.append-embedded-content|true|The document may have other embedded documents, for example, an Excel document may include a PDF content. If such an embedded content is available then, by default, it will be appended to the content of the master document, thus, in this example, the text extracted from PDF file will be appended to the text extracted from the Excel file. This property has to be set to `false` if one needs to access the content of the master and each of the embedded documents individually.  
+|quarkus.tika.tika-config-path||The resource path within the application artifact to the https://tika.apache.org/1.22/configuring.html[Tika configuration] resource. Typically a file named `tika-config.xml` is added to the root of the application resources path. The default configuration will be used if no configuration resource is available.
 |===
 
+== Parser configuration example
+
+[source]
+----
+quarkus.tika.parsers=pdf
+# Report the content in the order it occurs in the document
+# which can be useful for processing the invoices in PDF format 
+quarkus.tika.parser.pdf.sort-by-position=true
+----
+
+Please engage with Quarkus or Apache Tika communities to find out about a complete set of supported propertie for a given parser.
```

Comment:
I don't think this sentence makes sense? You expect users to ask the community what are the supported properties?

I think you need to provide some useful pointers.

Reply:
@gsmet I've given it hopefully a bit better wording. It is a bit tricky since I believe in most cases the users will be happy with just getting a text, customizing the parser requires some knowledge of how for example PDFBox works which Tika PDFParser delegates to and there is not that much info available specifically at the Tika site. And there could be so many parsers. 
So I've linked to the Tika source, alongside the suggestion to check with the community if required.
What do you think ?